### PR TITLE
fix(cf-deploy-config-writer): update destination policy and remove excess module

### PR DESCRIPTION
**Pull request by longieirl**
Fix for https://github.com/SAP/open-ux-tools/issues/2007

- the `existing_destinations_policy` should default to `update`
- the app content module shouldn't be added when creating a base config as this is only required when a HTML5 is being added
- `addDestinationService` is not required and is not referenced anywhere so is being removed

### CODE REVIEW

File: packages/cf-deploy-config-writer/src/cf-writer/base-config.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ async function addRoutingConfig(config: CFBaseConfig, fs: Editor): Promise<void>
     if (config.routerType === RouterModuleType.Standard) {
         await addStandaloneRouter(config, mtaConfigInstance, fs);
     } else {
-        await mtaConfigInstance.addRoutingModules(true);
+        await mtaConfigInstance.addRoutingModules(true, false);
```
Comment (kjose90) [inline packages/cf-deploy-config-writer/src/cf-writer/base-config.ts:56]: Thanks @longieirl 
PR is good and changes are clear.
A minor comment - would be nice to pass params as options here...
`options: { isManagedApp?: boolean; doCleanUp?: boolean } `

**Review by kjose90 — DISMISSED** [review]
Thanks @longieirl 
Approved ✅

File: packages/cf-deploy-config-writer/src/mta-config/mta.ts [hunk 2]
```diff
@@ -515,14 +516,17 @@ export class MtaConfig {
      * Append and/or cleanup the destination resource if missing in mta.yaml.
      *
      * @param {boolean} isManagedApp - if true, append managed approuter configuration
+     * @param {boolean} doCleanUp - if true, will ensure any missing modules | resources are appended
```
Comment (IainSAP) [inline packages/cf-deploy-config-writer/src/mta-config/mta.ts:519]: `doCleanUp` is a little ambiguous ? What about `addMissingModules`?

Comment (longieirl) [reply packages/cf-deploy-config-writer/src/mta-config/mta.ts:519]: Done

**Review by cianmSAP — APPROVED** [review]
Changes are clear
Changeset added
Not tested

**Review by kjose90 — APPROVED** [review]
Reapproving - Thanks!

**Pull request merged**